word_guess.rb

```diff
@@ -0,0 +1,159 @@
+class WordGuess
+  attr_accessor :solve_word, :number_of_lives
+    def initialize(hash)
+      @solve_word = hash[:solve_word]
+      @number_of_lives = hash[:number_of_lives]
+    end
+
+  def correct
+    puts "\nYes! You got one! No bunnies in your garden!"
+  end
+end
+
+#program set up
+welcome = <<YES
+
+Hello, gardner! We're going to play a game. I'm going to make you guess a
+word, and for each time you guess a letter incorrectly a new BUNNY is going to
+start to dig up your yard! HAHAHAHAHA!
+
+YES
+
+new_game = WordGuess.new(solve_word: "ruined", number_of_lives: 5)
+letter_array = new_game.solve_word.split(//)
+all_guesses = []
+count = new_game.number_of_lives
+
+alphabet_array = ["a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r", "s", "t", "u", "v", "w", "x", "y", "z"]
+
+unguessed_answer_letters_array = []
+unguessed_answer_letters = "__ " * letter_array.length
+unguessed_answer_letters_array = unguessed_answer_letters.split
+
+#begin program
+puts welcome
+puts "You have #{count} incorrect guesses before bunnies take over!"
+
+
+until count == 0
+  puts "You have already guessed: #{all_guesses}"
+  puts "\nHere's your word so far: #{unguessed_answer_letters_array.join(" ")}"
+  puts "\nGuess a letter and save your garden!"
+  guess = gets.chomp
+
+  until alphabet_array.include?(guess)
+  puts "Please enter a letter."
+    guess = gets.chomp
+  end
+
+if guess == letter_array[0]
```

Comment:
Watch your indentation here as well - this whole section should be indented as it is nested inside the `until`.